image_component.js

```diff
@@ -0,0 +1,8 @@
+

```

Comment:
Here is a small reference to how a react component looks:https://reactjs.org/docs/components-and-props.html